app/services/determine_sign_off_period.rb

```diff
@@ -1,15 +1,36 @@
 # frozen_string_literal: true
 
 class DetermineSignOffPeriod
-  CENSUS_MONTHS = [9, 10].freeze # September, October
-  PERFORMANCE_MONTH = 1 # January
+  VALID_PERIODS = %i[census_period performance_period outside_period].freeze
 
   def self.call
-    current_month = Time.zone.today.month
+    if Settings.sign_off_period.present?
+      if VALID_PERIODS.include?(Settings.sign_off_period.to_sym)
+        return Settings.sign_off_period.to_sym
+      else
+        Sentry.capture_exception(StandardError.new("Invalid sign_off_period value: #{Settings.sign_off_period}"))
+      end
+    end
```

Comment:
this method is getting a bit complex. how about extracting a method `valid_sign_off_period?`

```
def valid_sign_off_period?
  if VALID_PERIODS.include?(Settings.sign_off_period.to_sym)
    true
  else
    Sentry.capture_exception(StandardError.new("Invalid sign_off_period value: #{Settings.sign_off_period}"))
    false
  end
end
```

and then next to other `return` statements:

```
return Settings.sign_off_period.to_sym if Settings.sign_off_period.present? && valid_sign_off_period?
```

Reply:
agreed - I've moved it out to a method now